scalardb-sql-jdbc-sample/README.md

```diff
@@ -1,156 +1,174 @@
-# ScalarDB SQL (JDBC) Sample
+# Create a Sample Application That Uses ScalarDB SQL (JDBC)
 
-This tutorial describes how to create a sample application by using ScalarDB SQL (JDBC).
+{% capture notice--info %}
+**Note**
 
-## Prerequisites
+ScalarDB SQL (JDBC) is now deprecated. Please see [Getting Started with ScalarDB Cluster](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/getting-started-with-scalardb-cluster/) instead.
```

Comment:
Maybe it would be appropriate to state here: This sample code is now deprecated. To use ScalarDB SQL, you need to use ScalarDB Cluster. For more information about ScalarDB Cluster, See [ScalarDB Cluster document]().

As I commented above, the `scalardb-cluster` repository is private. So I think we need to link to the doc site.

Reply:
Thank you! I agree that a bit more information would be helpful here. How about the following?:

```suggestion
This sample code is now deprecated. To use ScalarDB SQL, you need to use ScalarDB Cluster, which is available only in the Enterprise edition. For more information, see [ScalarDB Cluster](https://scalardb.scalar-labs.com/docs/latest/scalardb-cluster/).
```

Reply:
LGTM!